shared/providers.ts

```diff
@@ -298,7 +305,11 @@ export function createHoverProvider(
             for await (const lsifResult of lsifProvider(textDocument, position)) {
                 if (lsifResult) {
                     await emitter.emitOnce('lsifHover')
-                    yield lsifResult
+                    if (hasPreciseResult) {
+                        yield lsifResult
+                    } else {
+                        yield { ...lsifResult, alerts: HoverAlerts.LSIF }
```

Comment:
Why are we not tagging the first lsif result?

Reply:
we're tagging *only* the first lsif result. is the var name confusing?

Reply:
Ah I see. Why not tag all results? I'm not sure how this is read by the webapp, but I'm assuming there's an implicit piece of knowledge about how it's displayed.

Reply:
The webapp does not ATM dedup alerts, so if every result has an alert on it it'll show multiple alerts. I agree there's probably a better way to do this, but not sure what it is. This is another artifact of designing through the extension API :(